backend/app/notify/email_test.go

```diff
@@ -0,0 +1,173 @@
+package notify
```

Comment:
tests should pass with the reace detector on, currenty it fails in TestEmailMultipleSend